Packs/DemistoRESTAPI/Scripts/GetIncidentTasks/GetIncidentTasks.yml

```
@@ -0,0 +1,67 @@
+commonfields:
+  id: GetIncidentTasks
+  version: -1
+name: GetIncidentTasks
+script: ''
+type: python
+subtype: python3
+tags:
+- Utility
+comment: Get all tasks for specific incident by the given state, name and/or tag.
+enabled: true
+args:
+- name: inc_id
+  required: true
+  default: true
+  description: Incident ID to get tasks from.
+- name: states
+  isArray: true
+  description: 'Comma separated list of states. Possible values: New, InProgress, Completed, Waiting, Error, Skipped, Blocked (leave empty to get all tasks).'
+- name: name
+  description: 'The name of the Task to search.'
+  type: string
+- name: tag
+  description: 'The tag to search.'
+  type: string
```

```suggestion
- name: tag
  description: 'The tag to search.'
```
It's not necessary to specify the `type` when the `type` is `string`